src/tzdata/build.jl

```diff
@@ -43,11 +43,7 @@ function build(
             # Retrieve the current latest version the cached latest has expired
             if latest_version === nothing
                 latest_version = last(tzdata_versions())
-                tzdata_hash = artifact_hash("tzdata$latest_version", ARTIFACT_TOML)
-
-                if tzdata_hash === nothing
-                    error("Latest tzdata is $latest_version which is not present in the Artifacts.toml")
-                end
+                @artifact_str "tzdata$latest_version"
```

Comment:
Did this need to be changed?

Reply:
It should have the same effect of throwing an error when the particular artifact does not exist, but by using `@artifact_str` the `Artifact.toml` contents gets embedded rather than looked up at runtime in the file, which may not exist if it's been relocated.